INSTALL.md

```diff
@@ -566,7 +566,7 @@ This option will be forced if on a platform that does not support CAPI.
 
 ### no-cmp
 
-Don't build support for Certificate Management Protocol (CMP).
+Don't build support for Certificate Management Protocol (CMP) and CRMF.
```

Comment:
Nit: for coherence should we expand the CRMF acronym?

Reply:
Good point - done.